boards/xtensa/esp32s3/common/include/esp32s3_board_spislavedev.h

```diff
@@ -1,5 +1,5 @@
 /****************************************************************************
- * boards/xtensa/esp32s2/common/include/esp32s2_board_spislavedev.h
+ * boards/xtensa/esp32s3/common/include/esp32s3_board_spislavedev.h
```

Comment:
merge this patch to:
https://github.com/apache/nuttx/pull/16866/commits/4bb93d3539033e5fa9bfccb06d49e6d9e4e9b2e4